src/x509_cert_generator.ts

```diff
@@ -37,6 +37,10 @@ export interface X509CertificateCreateParamsBase {
    * Signing algorithm
    */
   signingAlgorithm: Algorithm | EcdsaParams;
+  /**
+   * Signature for manually initialized certificates
+   */
+  signature?: Buffer;
```

Comment:
`@peculiar/x509` is a cross-platform package. `Buffer` is NodeJS type. Use [SourceBuffer](https://www.w3.org/TR/WebIDL-1/#common-BufferSource) type instead